### PR TITLE
Elasticsearch on aws

### PR DESCRIPTION
**Pull request by DRoyyy**

### CODE REVIEW

File: content/post/run-elasticsearch-in-aws/index.md [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+```bash
+$ kubectl get elasticsearch -n demo es-topology-cluster
+NAME                  VERSION       STATUS   AGE
+es-topology-cluster   xpack-8.2.0   Ready    3m1s
+```
+> We have successfully deployed Elasticsearch in Amazon EKS. Now we can exec into the container to use the database.
+
+### Insert Sample Data
+
+In this section, we are going to create few indexes in Elasticsearch. At first, we are going to port-forward the respective Service so that we can connect with the database from our local machine. Then, we are going to insert some data into the Elasticsearch.
```
Comment (raihankhan) [inline content/post/run-elasticsearch-in-aws/index.md:276]: On the deployment of Elasticsearch yaml, the operator creates a governing service that is named after the Elasticsearch object name itself. We are going to use this object to....

File: content/post/run-elasticsearch-in-aws/index.md [hunk 2]
```diff
@@ -0,0 +1,692 @@
...
+    "successful" : 2,
+    "failed" : 0
+  },
+  "_seq_no" : 0,
+  "_primary_term" : 1
+}
+
+```
+
+Now, let’s verify that the index have been created successfully.
```
Comment (raihankhan) [inline content/post/run-elasticsearch-in-aws/index.md:356]: has been

File: content/post/run-elasticsearch-in-aws/index.md [hunk 3]
```diff
@@ -0,0 +1,692 @@
...
+    ]
+  }
+}
+
+```
+
+
+> You can see the database has been restored. The recovery of Elasticsearch has been successful. If you faced any difficulties in the recovery process, you can reach out to us through [EMAIL](mailto:support@appscode.com?subject=Stash%20Recovery%20Failed%20in%20AWS).
+
+We have made an in depth video on Elasticsearch Hot-Warm-Cold Architecture Management with Kibana in Kubernetes Using KubeDB. You can have a look into the video below:
```
Comment (raihankhan) [inline content/post/run-elasticsearch-in-aws/index.md:676]: We have an in-depth walkthrough on...

**Review by raihankhan — CHANGES_REQUESTED** [review]

**Review by raihankhan — APPROVED** [review]
LGTM!

**Review by kamolhasan — APPROVED** [review]

**Pull request merged**